### PR TITLE
hash is more portable in bash than command -v

### PR DESCRIPTION
**Pull request by chayim**
also fixing minor check in py3

### CODE REVIEW

File: bin/getpy3 [hunk 1]
```diff
@@ -71,7 +71,7 @@ check_variants() {
 
 	export MINOR="$($MYPY --version 2>&1 | awk '{print $2}' | cut -d. -f2)"
 	# Python3 <= 3.4 breaks contemporary pip
-	if (( MINOR < 5 )); then export FORCE=1; fi
+	if (( ${MINOR} < 5 )); then export FORCE=1; fi
```
Comment (rafie) [inline bin/getpy3:74]: Won't `(( MINOR < 5 ))` work as well? (vars get expanded inside (()), don't they?

File: shibumi/functions [hunk 2]
```diff
@@ -107,7 +107,7 @@ fixterm() {
 #----------------------------------------------------------------------------------------------
 
 is_command() {
-        command -v $1 &> /dev/null
+    hash $1 &>/dev/null
```
Comment (rafie) [inline shibumi/functions:110]: Is `hash` considered a better practice than `command`? Is there caching involved in this?

Comment (chayim) [reply shibumi/functions:110]: That, and because its older.

File: bin/getpy3 (hunk 1 above)
Comment (chayim) [reply bin/getpy3:74]: Correct, but we want to expand the variable. We want to ensure that the value of MINOR is < 5 as opposed to the string MINOR.

**Pull request closed without merging**